pavelib/paver_tests/test_servers.py

```diff
@@ -297,6 +302,7 @@ def verify_run_all_servers_task(self, options):
                 node_env="production",
                 static_root_lms=None,
                 static_root_cms=None,
+                js_env_extra_config=json.dumps({}),
```

Comment:
According to the [comment raised earlier](https://github.com/edx/edx-platform/pull/25695/files#r553657732), update the `json.dumps({})` to `json.dumps("{}")` for consistency 👍🏼

Reply:
Addressed 👍🏼